filterPackages.nix

```
@@ -0,0 +1,27 @@
+system: packages:
+let
+  # Adopted from nixpkgs.lib
+  inherit (builtins) listToAttrs concatMap attrName;
+  nameValuePair = name: value: { inherit name value; };
+  filterAttrs = pred: set:
+    listToAttrs (
+      concatMap (name:
+        let v = set.${name}; in
+          if pred name v then [(nameValuePair name v)] else []
+      )
+      (attrNames set)
+    );
+
+  # Everything that nix flake check requires for the packages output
+  sieve = (n: v: with v;
```

```suggestion
  sieve = n: v:
    with v;
```
The parenthesis is not needed here.